Items/07_Items.txt

```diff
@@ -32,18 +32,31 @@ Engineer processes; i.e., a Gunslinger who buys an extruder can't construct
 IEDs with it.
 
 3D image builder, Static ($650):
-    A 3m^3 framework that will build a rudimentary  3d image of whatever is 
+    A 3m^3 framework that will build a rudimentary 3d image of whatever is 
 inside it, exportable to a portable device.
 
 3D image builder, Portable ($800):
     Builds a 3D image of what its being pointed at, will need to see the object 
 from multiple angles to capture the whole object. Up to 20m^3 of space.
-
-Medical Nanites Unit ($):
-	One unit of Medical Nanites.
 	
-Medical Nanites Booster Shot ($):
-	Half a unit of Medical Nanites.
+Medical Nanites Booster Shot ($400):
+	A partial unit of Medical Nanites. The booster shot is a standard 
+medical procedure on more affluent worlds, like a vaccine, and provides 
+resistance to and faster healing from disease and injury. Characters with a 
+nanite booster shot gain +10 to Shock saves and benefit from faster healing 
+during Short Rests (see Resting in the Basic Rules).
+	Additional boosters after the first have no effect. A booster shot 
+may be upgraded to a Medical Nanites Unit (see below) at a cost of $700.
+	
+Medical Nanites Unit ($1000):
+	One unit of Medical Nanites. Having at least one unit of Medical Nanites 
+provides all the benefits of a booster shot (see above), with even more potent 
+rest healing. Additionally, full units of medical nanites can be harnessed to 
+execute Active Feats (see Feats in the Basic Rules).
+	Additional units after the first do not provide more healing or increase 
+the save bonus, but may be used to execute more active feats in shorter time 
+frames. The maximum number of Units a character can have at once is related to 
+their level - see the leveling table.
```

Comment:
Blocker. This is a really good idea, but I think we need to add edge-case guard rails. In formal terms, this allows the linear-scale buying of power without depreciating marginal gains. Or in laymen's terms, what happens if someone sells a small spaceship and dumps all of that money into Med Nanites. 
One potential guard rail would be a by-level maximum which is explained diajetically that introducing too many nanites to your system at once is really hard on your immune system. Another guard rail could be a limit to only being able to get one more unit per level.

Reply:
I hadn't considered waiting until level 8 and dumping three units into yourself all at once. I would want to add acquisition rate-limits based on flavor alone.
Are you also asking me to make units after the first cost more?